app/src/lib/editors/win32.ts

```diff
@@ -75,7 +76,8 @@ function isExpectedInstallation(
       return displayName === 'Atom' && publisher === 'GitHub Inc.'
     case ExternalEditor.VisualStudioCode:
       return (
-        displayName === 'Visual Studio Code' &&
+        (displayName === 'Visual Studio Code' ||
+          displayName === 'Microsoft Visual Studio Code Insiders') &&
```

Comment:
I think this is going to error out because I initially chose the simpler value:

![](https://user-images.githubusercontent.com/359239/31580119-81cff5a4-b192-11e7-80f5-be48e28086c5.PNG)

Not sure if we need to change this, but I don't think we need to look for `Microsoft` here.

Reply:
Yep, on my machine the 32-bit insider build has `Inno Setup: Icon Group` has value `Visual Studio Code - Insiders`. Let's use that.

Reply:
Fixed @shiftkey